README/ReleaseNotes/v624/index.md

```diff
@@ -126,6 +128,14 @@ See [Demo notebook in SWAN](https://github.com/hageboeck/rootNotebooks),
 [EPJ Web Conf. 245 (2020) 06007](https://www.epj-conferences.org/articles/epjconf/abs/2020/21/epjconf_chep2020_06007/epjconf_chep2020_06007.html),
 [arxiv:2012.02746](https://arxiv.org/abs/2012.02746).
 
+### RooBatchCompute Library
+A new library has been created with the aim of concentrating the auto-vectorizable compute functions of roofit,
+as well as all the helpers (classes, structs, helping functions) that provide this functionality.
+
+In case you have created a custom PDF which uses the batch computations feature, or you want to introduce this feature to your PDFs for the first time,
+you need to override [`RooAbsReal::evaluateSpan()`](https://root.cern.ch/doc/master/classRooAbsReal.html#a1e5129ffbc63bfd04c01511fd354b1b8) and implement 
+one computing function inside the RooBatchCompute Library, under the directory `roofit/batchcompute`. You can use any vectorised PDF inside this library as
+an example on how to do this, and/or consult the [RooBatchCompute readme](roofit/batchcompute/README.md).
```

Comment:
I'm not sure this link is going to work, e.g. when viewing the release notes in a browser. Maybe it's better to unlink it (and describe where to find the file) or give the full `https://root.cern/...`

Reply:
It might still be difficult to find out how to migrate.

Instead of `evaluateBatch(size_t, size_t)`, and using something like `x.getValBatch(start, end)` on a `RooRealProxy`, one now has to retrieve the computation data using `x.getValues(runContext, normSet)`. Can you e.g. describe this transition using two code examples, for example comparing the Gaussian from 6.22 with the new method?

Reply:
Gaussian is actually a bad example. Here is `RooAddPdf`:
```diff
 ////////////////////////////////////////////////////////////////////////////////
 /// Compute addition of PDFs in batches.
-
-RooSpan<double> RooAddPdf::evaluateBatch(std::size_t begin, std::size_t batchSize) const {
-  auto normAndCache = getNormAndCache();
+RooSpan<double> RooAddPdf::evaluateSpan(RooBatchCompute::RunContext& evalData, const RooArgSet* normSet) const {
+  auto normAndCache = getNormAndCache(normSet);
   const RooArgSet* nset = normAndCache.first;
   CacheElem* cache = normAndCache.second;
 
 
-  auto output = _batchData.makeWritableBatchInit(begin, batchSize, 0.);
-  const std::size_t n = output.size();
-
+  RooSpan<double> output;
 
   for (unsigned int pdfNo = 0; pdfNo < _pdfList.size(); ++pdfNo) {
     const auto& pdf = static_cast<RooAbsPdf&>(_pdfList[pdfNo]);
-    auto pdfOutputs = pdf.getValBatch(begin, batchSize, nset);
-    assert(pdfOutputs.size() == output.size());
+    auto pdfOutputs = pdf.getValues(evalData, nset);
+    if (output.empty()) {
+      output = evalData.makeBatch(this, pdfOutputs.size());
+      for (double& val : output) { //CHECK_VECTORISE
+        val = 0.;
+      }
+    }
+    assert(output.size() == pdfOutputs.size());
 
     const double coef = _coefCache[pdfNo] / (cache->_needSupNorm ?
         static_cast<RooAbsReal*>(cache->_suppNormList.at(pdfNo))->getVal() :
         1.);
 
     if (pdf.isSelectedComp()) {
-      for (std::size_t i = 0; i < n; ++i) { //CHECK_VECTORISE
+      for (std::size_t i = 0; i < output.size(); ++i) { //CHECK_VECTORISE
         output[i] += pdfOutputs[i] * coef;
       }
     }
```